pyfstat/mcmc_based_searches.py

```diff
@@ -1233,37 +1233,10 @@ def plot_cumulative_max(self, **kwargs):
 
         if hasattr(self, "search") is False:
             self._initiate_search_object()
-        if self.binary is False:
-            self.search.plot_twoF_cumulative(
-                self.label,
-                self.outdir,
-                F0=d["F0"],
-                F1=d["F1"],
-                F2=d["F2"],
-                Alpha=d["Alpha"],
-                Delta=d["Delta"],
-                tstart=self.minStartTime,
-                tend=self.maxStartTime,
-                **kwargs,
-            )
-        else:
-            self.search.plot_twoF_cumulative(
-                self.label,
-                self.outdir,
-                F0=d["F0"],
-                F1=d["F1"],
-                F2=d["F2"],
-                Alpha=d["Alpha"],
-                Delta=d["Delta"],
-                asini=d["asini"],
-                period=d["period"],
-                ecc=d["ecc"],
-                argp=d["argp"],
-                tp=d["argp"],
-                tstart=self.minStartTime,
-                tend=self.maxStartTime,
-                **kwargs,
-            )
+
+        self.search.plot_twoF_cumulative(
+            CFS_input=d, label=self.label, outdir=self.outdir, **kwargs
+        )
```

Comment:
This call makes sense now.

Just for confirmation  @Rodrigo-Tenorio : end user is expected to pass `PFS_input` through `kwargs` if they e.g. want to compare the loudest candidate vs injected values?

If yes, since that should be a fairly popular use case, we should mention it explicitly in the documentation of this method.

Reply:
Yes, every other argument (including `PFS_input` or even `savefig` has to be passed through kwargs.

Reply:
Ok. On second thought, let's not bother with amending the doc, it points clearly enough to the core function.

Reply:
I'd say something like  57833ee is explicit enough?

Reply:
Oh, you're fast! Perfect.

Reply:
Tbh, that doc was pointing to a `pyfstat.core.plot_twoF_cumulative`, which doesn't exist anymore as a function (does this mean PyFstat started as a bunch of functions and then it got objectified? anyways, that's for another time).